# 硬件配置文件
hardware:
  # 硬件配置列表
  single_arm:               # 单臂时为"single_arm"，双臂时可设为"left_arm"或"right_arm"
      robot_type: "arm620"
      interface: can0
      motors: [1, 2, 3, 4, 5, 6]    # 电机ID列表
      joint_names: ["joint1", "joint2", "joint3", "joint4", "joint5", "joint6"]
      frame_id: "base_link"
      controller_name: "arm_controller"  # 对应的控制器名称
      planning_group: "arm"         # MoveIt规划组名称
      initial_position: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]  # 初始位置
      start_position: [0.0, -0.5236, -0.7854, 0.0, 0.5236, 0.0]  # 启动位置

  # left_arm_mapping:
  #     robot_type: "arm620"
  #     interface: can0
  #     motors: [1, 2, 3, 4, 5, 6]    # 电机ID列表
  #     joint_names: ["left_joint1", "left_joint2", "left_joint3", "left_joint4", "left_joint5", "left_joint6"]
  #     frame_id: "left_base_link"
  #     controller_name: "left_arm_controller"  # 对应的控制器名称
  #     planning_group: "left_arm"         # MoveIt规划组名称
  #     initial_position: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]  # 初始位置
  #     start_position: [0.0, -0.5236, -0.7854, 0.0, 0.5236, 0.0]  # 启动位置
  # right_arm_mapping:
  #     robot_type: "arm620"
  #     interface: can1
  #     motors: [1, 2, 3, 4, 5, 6]    # 电机ID列表
  #     joint_names: ["right_joint1", "right_joint2", "right_joint3", "right_joint4", "right_joint5", "right_joint6"]
  #     frame_id: "right_base_link"
  #     controller_name: "right_arm_controller"  # 对应的控制器名称
  #     planning_group: "right_arm"         # MoveIt规划组名称
  #     initial_position: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]  # 初始位置
  #     start_position: [0.0, 0.5236, 0.7854, 0.0, -0.5236, 0.0]  # 启动位置
